### PR TITLE
[chore] fix typo in resource attributes docs

### PR DESCRIPTION
**Pull request by SylvainJuge**
## Changes

This fixes what looks like a typo in resource attributes documentation for the `telemetry.distro.*` group.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] ~~Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).~~ changelog entry not needed
  * If your PR does not need a change log, start the PR title with `[chore]`
* [x] ~~[schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.~~ not needed.

### CODE REVIEW

File: docs/resource/README.md [hunk 1]
```diff
@@ -186,11 +186,11 @@ All custom identifiers SHOULD be stable across different versions of an implemen
...
 
-## Telemetry SDK (Experimental)
+## Telemetry Distribution (Experimental)
 
 **Status**: [Experimental][DocumentStatus]
 
-**type:** `telemetry.sdk`
+**type:** `telemetry.distro`
 
 **Description:** Additions to the telemetry SDK.
```
Comment (lmolkova) [inline docs/resource/README.md:195]:
```suggestion
**Description:** The telemetry distribution (distro) used to capture data recorded by the instrumentation libraries.
```

**Review by lmolkova — APPROVED** [review]
/cc @jsuereth @tigrannajaryan

File: docs/resource/README.md [hunk 2]
```diff
@@ -186,11 +186,11 @@ All custom identifiers SHOULD be stable across different versions of an implemen
 | `webjs` | webjs | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 <!-- endsemconv -->
 
-## Telemetry SDK (Experimental)
+## Telemetry Distribution (Experimental)
 
 **Status**: [Experimental][DocumentStatus]
 
-**type:** `telemetry.sdk`
+**type:** `telemetry.distro`
```
Comment (lmolkova) [inline docs/resource/README.md:193]: not related to this PR, feel free to resolve if necessary - I wonder what type means in scope of this document,

**Comment by tigrannajaryan** [comment]
Nit: both `telemetry.sdk.*` and `telemetry.distro.*` [attributes here are in one section](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/telemetry.md) called `Telemetry SDK`. Do we want them in separate sections in docs/resource/README.md? I don't have a strong opinion, works either way for me.

**Review by AlexanderWert — APPROVED** [review]

**Comment by SylvainJuge** [comment]
> Nit: both `telemetry.sdk.*` and `telemetry.distro.*` [attributes here are in one section](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/telemetry.md) called `Telemetry SDK`. Do we want them in separate sections in docs/resource/README.md? I don't have a strong opinion, works either way for me.

What do you suggest here, having a single `Telemetry` section instead with all the `telemetry.*` attributes ?
I can definitely do that in a follow-up PR once this is merged (just :+1: on this comment and I'll do that).

File: docs/resource/README.md (hunk 2 above)
Comment (pyohannes) [reply docs/resource/README.md:193]: > I wonder what type means in scope of this document.

Good question. I'm not sure, but I'd guess it means that resource attributes of the same type should be provided by the same resource detector?

**Review by pyohannes — APPROVED** [review]

File: docs/resource/README.md (hunk 2 above)
Comment (joaopgrassi) [reply docs/resource/README.md:193]: No idea what the type here is about, seems totally made up 🤔.

Comment (lmolkova) [reply docs/resource/README.md:193]: Created https://github.com/open-telemetry/semantic-conventions/issues/929 to follow up

**Pull request merged**